### DISCUSSION TITLE
Request: Support redirect at the proxy level

### DISCUSSION THREAD

**Discussion by guillaumebriday** [basecamp/kamal-proxy#42] [Ideas and Issue Triage]
It would be great if Kamal-proxy could manage redirects at the proxy level.

A key example would be redirecting non-www to www, or the other way around.

Here's how we handled it with Traefik: https://lesscodeworks.com/redirect-from-www-to-non-www-with-kamal-and-traefik  
And here's an example for doing it with Rails: https://masilotti.com/rails-redirect-www/

What do you think about it? How do you handle it at basecamp?

**Comment by khaled-badenjki** [comment]
Until this is implemented, if you're using cloudflare for your website (and you should), you can use their page rules to create such behavior. here is [how](https://community.cloudflare.com/t/redirect-the-www-to-non-www/558655)
